git-squash-tags

```diff
@@ -8,16 +8,18 @@ if [ $# -eq 1 ]; then
 fi
 
 echo "Grabbing tags following pattern $tag_prefix..."
-tags=(`git log --reverse --pretty=%d --tags=$tag_prefix --no-walk | sed 's/ (tag: //' | sed 's/)//'`)
+tags=(`git log --reverse --pretty=%d --tags=$tag_prefix --no-walk |grep 'tag:' | sed 's/.*tag: //' | sed 's/)//'|sed 's/,.*//'`)
 
 # pull out first tag and remove it from $tags as it will be used as base
 tags_first=${tags[0]}
 tags=${tags[@]:1}
 echo "Using $tags_first as base."
+NEWLINE=$'\n'
 
 function commitAll {
   git add --all .
-  git commit -m "Squashed $tag."
+  log=`git log $previous..$tag`
+  git commit -m "Squashed $tag$NEWLINE$NEWLINE$log"
```

Comment:
This changes the log message to be much more verbose which is an entirely different discussion than fixes the bug indicated by the title.